pypdf/_writer.py

```diff
@@ -920,7 +920,8 @@ def _update_text_field(self, field: DictionaryObject) -> None:
         else:  # /Tx
             txt = field.get("/V", "")
             sel = []
-
+        # Escape parentheses (Ref. 7.3.4.2 Literal Strings)
+        txt = txt.replace('(', r'\(').replace(')', r'\)')
```

Comment:
\ should also be covered
```suggestion
        txt = txt.replace("\\", "\\\\").replace("(", r"\(").replace(")", r"\)")
```